### DISCUSSION TITLE
How to intercept user registration

### DISCUSSION THREAD

**Discussion by andy250** [logto-io/logto#5489] [Q&A]
User registers with LogTo, then I want to intercept that event and:
* add user to my database (I know, LogTo suggests not doing it)
* send the user to "Setup your organization" web page (every new user will be a new organization)
* later users can be added to organizations only via an invite from within the organization - is there anything built into LogTo that supports it?

**Comment by wangsijie** [comment]
It's a good idea to enable running custom code scripts for specific scenarios in Logto.

Currently, webhook + management API is a work around.
